scripts/publishToAttestationStation.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs')
+const csvParser = require('csv-parser')
+const ethers = require('ethers')
+const hardhat = require('hardhat')
+const attestationStationAbi = require('./abi/AttestationStationABI.json')
+const attestationStationAddress = '0xEE36eaaD94d1Cc1d0eccaDb55C38bFfB6Be06C77'
+
+const csvFilePath = './scripts/data/attestations.csv'
+
+const parseCsv = async filePath => {
+  const results = []
+  return new Promise((resolve, reject) => {
+    fs.createReadStream(filePath)
+      .pipe(csvParser())
+      .on('data', data => results.push(data))
+      .on('end', () => {
+        resolve(results)
+      })
+      .on('error', error => {
+        reject(error)
+      })
+  })
+}
+
+const createAttestations = async csvData => {
```

Comment:
how is this script run? can we document in readme, please? 

And then could we read the csvFilePath from the command line as an argument? That way lukas can run this script by himself?

Reply:
To start we had 1800 rows in the CSV.

The first time I ever ran the script, I ran it 9 times with 200 rows. For future runs of the script we need to only publish the scores that updated (otherwise we would be paying for updating scores that didn't change)

I made a script that handles this and added documentation for how to run it in the comments at the top of the file

Reply:
so, the workflow needs to be
- export csv from google sheets
- import it to the data folder
- run the update_scores.py file
- run the publishAttestations function